packages/sdk/src/conversation/interface.ts

```diff
@@ -261,6 +301,60 @@ export interface TeamsFxAdaptiveCardActionHandler {
   handleActionInvoked(context: TurnContext, actionData: any): Promise<InvokeResponse>;
 }
 
+/**
+ * Interface for SSO configuration for Bot SSO
+ */
+export interface SsoConfig {
+  /**
+   * The list of scopes for which the token will have access
+   */
+  scopes: string[];
+
+  /**
+   * Custom sso execution activity handler class which should implement the interface {@link SsoExecutionActivityHandler}. If not provided, it will use {@link DefaultSsoExecutionActivityHandler} by default
+   */
+  CustomSsoExecutionActivityHandler?: new (ssoConfig: SsoConfig) => SsoExecutionActivityHandler;
+
```

Comment:
The name may be too common, since npm package does not support namespace, we may use a more spefic name to avoid confusing?

Reply:
Will discuss with PM to finalize the name